components/Project.js

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+
+export default function Project({ project }) {
+  if (!project) return;
+
+  const { _id: id, title } = project;
+
+  return (
+    <li>
+      <Link href={`/projects/${id}`}>
+        <h3>{title}</h3>
```

Comment:
"A common navigation technique for users of screen reading software is to quickly jump from heading to heading in order to determine the content of the page. Because of this, it is important to not skip one or more heading levels" please read again [about headings ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de) ;)